subhub/tests/unit/test_payment_calls.py

```diff
@@ -362,11 +369,14 @@ def test_cancel_subscription_with_valid_data_multiple_subscriptions_remove_secon
 
 def test_cancel_subscription_with_invalid_data(app, create_subscription_for_processing):
     (subscription, code) = create_subscription_for_processing
-    (cancelled, code) = payments.cancel_subscription(
-        "process_test", subscription["subscriptions"][0]["subscription_id"] + "invalid"
-    )
-    assert cancelled["message"] == "Subscription not available."
-    assert 400 == code
+    logger.info("subscription", data=subscription)
+    if subscription.get("subscriptions"):
+        (cancelled, code) = payments.cancel_subscription(
```

Comment:
why the tuple () parens?

Reply:
This is a valid return to create a tuple, however it is unneeded, so I removed from this line and the remainder of the file.